### DISCUSSION TITLE
Conditional Navigating to different pages

### DISCUSSION THREAD

**Discussion by HaHa-Clint** [Budibase/budibase#11742] [Help]
Is it possible to have a conditional navigation to different pages? 

I have a page where the customer will select their method of payment whether that be: Invoice, Credit Card, or Check. 

Now based on the option my customer selected, I need to navigate to the appropriate page for them to log their Credit card information, Check information or Invoice information.

Is this possible using the Budibase DB. 

Thank you for the help.

**Comment by ConorWebb96** [comment]
Hey @HaHa-Clint,

If you are taking about the navigation which can be set across all the pages. Then the answer at this moment is no, this is due to the only conditions that can be set on these being role related permissions.

Now with that being said you could hide the default Budibase navigation and create your own with containers and links/buttons. These elements can then have some conditional ui or amending with dynamic bindings which should do what you are wanting. 

Below I have provided a very basic app export that will demo how this can be done with a **form, button component and bindings**. Keep in mind that i did this on the latest release of Budibase.

[nav-change-demo.gz](https://github.com/Budibase/budibase/files/12586488/nav-change-demo.gz)

I hope this helps.
